test/cases/api.sh

```diff
@@ -499,6 +499,14 @@ OUTPUT=$(curl \
 
 COMPOSE_ID=$(echo "$OUTPUT" | jq -r '.id')
 
+METRICS_OUTPUT=$(curl \
+  --cacert /etc/osbuild-composer/ca-crt.pem \
+  --key /etc/osbuild-composer/client-key.pem \
+  --cert /etc/osbuild-composer/client-crt.pem \
+  https://localhost/metrics)
+
+PREV_COMPOSES=$(echo "$METRICS_OUTPUT" | grep "^total_compose_requests" | cut -f2 -d' ')
```

Comment:
As the first `/metrics` curl happens after the first `/compose` request, the `total_compose_requests` is already 1; so the test later on will fail as `$((PREV_COMPOSES + 1))` is 2.

So just move the first /`metrics` call to before the first `/compose` call and you should be good. Mind if you also check the `"^total_successful_compose_requests"` ? Just for completeness, check it in the same way as PREV_COMPOSES, but then like PREV_SUCCESSFUL_COMPOSES or something.